compaction/compaction_manager.hh

```diff
@@ -569,7 +569,7 @@ protected:
                                 sstables::offstrategy offstrategy = sstables::offstrategy::no);
     future<> update_history(::compaction::compaction_group_view& t, compaction_result&& res, const compaction_data& cdata);
     bool should_update_history(compaction_type ct) {
-        return ct == compaction_type::Compaction;
+        return ct == compaction_type::Compaction || ct == compaction_type::Major;
```

Comment:
this is a (preexisting) bug. after we silenced compaction logs, it would be important to have all types updating history. we gained the ability to record the type and origin (shard).

/cc @avikivity

Reply:
@raphaelsc I created an issue to track this - https://github.com/scylladb/scylladb/issues/26800

Reply:
thanks